src/kOS/Binding/BindingManager.cs

```diff
@@ -137,18 +137,14 @@ public void ToggleFlyByWire(string paramName, bool enabled)
 
         public void UnBindAll()
         {
-            if (flightControl != null)
-            {
-                flightControl.UnBind();
-            }
+            // TODO: Remove this method
+            // Do nothing, left over from old steering logic
         }
 
         public void Dispose()
         {
-            if (flightControl != null)
-            {
-                flightControl.Dispose();
-            }
+            // TODO: Remove this method
+            // Do nothing, left over from old steering logic
         }
```

Comment:
Probably not.  I wanted to make sure I didn't loose the reference to all the locations that called these methods.  I'm not convinced that I actually want to remove this logic, and it may be more tied to your requested suffix to control how kOS releases the controls on program end.  Based on a couple of things you reported as a bug with steering/throttle releasing after program termination, I think I want to re-add the unbind logic.